conans/test/util/files_test.py

```diff
@@ -33,3 +35,13 @@ def modified_only_true_test(self):
     def modified_only_false_test(self):
         save(self.filepath, "other content", only_if_modified=False)
         self.assertNotEqual(self.timestamp, os.path.getmtime(self.filepath))
+
+    def walk_encoding_test(self):
+        badfilename = "\xE3\x81\x82badfile.txt"
+        folder = temp_folder()
+        filepath = os.path.join(folder, badfilename)
+        if six.PY2:
+            folder = unicode(folder)
```

Comment:
Maybe use `six.u(folder)` in line 46 and avoid this if clause?

Reply:
Interesting this ``six.u``, I didn't now about it.